src/coreclr/gc/vxsort/isa_detection.cpp

```diff
@@ -19,28 +19,6 @@ enum class SupportedISA
 
 #if defined(TARGET_AMD64) && defined(TARGET_WINDOWS)
 
-static DWORD64 GetEnabledXStateFeaturesHelper()
-{
-    // On Windows we have an api(GetEnabledXStateFeatures) to check if AVX is supported
-    typedef DWORD64(WINAPI* PGETENABLEDXSTATEFEATURES)();
-    PGETENABLEDXSTATEFEATURES pfnGetEnabledXStateFeatures = NULL;
-
-    HMODULE hMod = LoadLibraryExW(L"kernel32.dll", NULL, LOAD_LIBRARY_SEARCH_SYSTEM32);
-    if (hMod == NULL)
-        return 0;
-
-    pfnGetEnabledXStateFeatures = (PGETENABLEDXSTATEFEATURES)GetProcAddress(hMod, "GetEnabledXStateFeatures");
```

Comment:
https://github.com/dotnet/runtime/blob/08a6e06d8fdc29efebc5c0ec7d2b6ba215810e71/src/coreclr/vm/threadsuspend.cpp#L1972 links to `GetEnabledXStateFeatures` directly, so it is safe to delete the dynamic linking here.

Reply:
It's worth noting that the original reason for the dynamic linking was to support Windows 7 prior to SP1.

With Win7 support having been dropped and us just not regressing Win 7 SP1 and later, this is fine.